app/Http/Controllers/Traits/TagsRequests.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rogue\Http\Controllers\Traits;
+
+use Rogue\Http\Requests\TagsRequest;
+
+trait TagsRequests
+{
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(TagsRequest $request)
+    {
+        $post = $this->post->find($request->post_id);
```

Comment:
It's a little awkward that this has an implicit dependency on the controller's `post` field. Not sure how best to improve that, but just food for thought.